packages/terra-data-grid/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -196,8 +254,8 @@ const ColumnHeaderCell = (props) => {
       role="columnheader"
       scope="col"
       aria-sort={sortIndicator}
-      onMouseDown={(isSelectable && onColumnSelect) ? handleMouseDown : undefined}
-      onKeyDown={(isSelectable && onColumnSelect) ? handleKeyDown : undefined}
+      onMouseDown={onColumnSelect ? handleMouseDown : null}
```

Comment:
Why `null` and not `undefined` like we have before? I believe if these are set `null` then these will show up in  snapshots versus undefined.

Reply:
Yeah, you're right. I originally changed them for consistency but we don't want these props to show up if they are not used. Reverted them back here: b00d53fbb56b9221531f03703da298489563f9d4